DeployWithLibraries/1_Writing_a_Library/1_contractSimpleLibrary.sol

```diff
@@ -5,3 +5,5 @@ contract test {
             // should call a library method which returns `3`
     }
 }
+
+// Put the library here
```

Comment:
This is an indication for the test?

Reply:
this is to help a student know where to put the library.  Or at least so it will match the answer file.